src/components/Base/CodeMirror/index.jsx

```diff
@@ -21,21 +20,31 @@ export default class CodeMirrorX extends React.Component {
     mode: 'yaml'
   };
 
+  constructor(props) {
+    super(props);
+    this.state = { module: null };
+  }
+
   componentDidMount() {
-    this.refs.editor.getCodeMirror().refresh();
+    import('react-codemirror').then(module => this.setState({ module: module.default }));
```

Comment:
Since we upgrade to react v16.6. You can refactor like this:
```
const CodeMirror = React.lazy(()=> import('/path/to/react-codemirror'))
```

Reply:
ok